powersimdata/input/input_data.py

```diff
@@ -99,7 +99,7 @@ def get_profile_version(self, scenario_info, kind):
         query = posixpath.join(
             server_setup.DATA_ROOT_DIR,
             server_setup.BASE_PROFILE_DIR,
-            scenario_info["grid_model"],
```

Comment:
Would we want to switch back to this way once the scenario list is updated to include the new column?

Reply:
I don't think so. The `get_profile_version` method of the `InputData` class just needs the`grid_model`. We could still pass the `scenario_info` dictionary that has `grid_model` has a key or produce a one item dictionary when the `scenario_info` dictionary is not available e.g.  [here](https://github.com/Breakthrough-Energy/PowerSimData/blob/2ee61845ead64f16dc2d1fb8d21dcd8831c3566e/powersimdata/scenario/create.py#L339).

I guess it would be nice to mirror what we have in the `get_data` method of the same class. There, it makes sense to pass the `scenario_info` dictionary because it requires the `id`, `grid_model` and the version of the profile (e.g. `base_demand`).

Let me know what you think

Reply:
I see what you mean. Another possibility would be passing the `scenario_info` in the `InputData` constructor,  but I'm not sure if that's better. I'd keep it the way you have it now since that seems reasonable.

Reply:
Yeah. Not super thrilled about it because i will have to refactor all the modules that instantiate the `InputData` class. We could do that in a different PR if necessary.